runtime/oti/j9consts.h

```diff
@@ -276,7 +276,7 @@ extern "C" {
 #define J9_RUNTIME_SHUTDOWN 0x100
 #define J9_RUNTIME_XFUTURE 0x200
 #define J9_RUNTIME_PAINT_STACK 0x400
-#define J9_RUNTIME_AOT_STRIPPED 0x800
+#define J9_RUNTIME_UNUSED 0x800
```

Comment:
The naming convention for flags like this includes the value, i.e. `J9_RUNTIME_UNUSED_0x800`. Same comment for the unused below.

Reply:
fixed